src/integration/tests/firstTest.test.tsx

```diff
@@ -76,123 +150,498 @@ describe("First test", () => {
+
+    expect(alertSpy).toHaveBeenCalledTimes(1);
+    const valueNegativeError = "Valor inválido!";
+
+    expect(alertSpy).toHaveBeenCalledWith(valueNegativeError);
+    alertSpy.mockRestore();
+  });
+
+  it("should not be possible to insert an item without the fields date, category, title and value", () => {
+    const alertSpy = jest.spyOn(window, "alert").mockImplementation(() => {});
+
+    const inputArea = within(screen.getByTestId(INPUT_AREA_ID));
+    const buttonAddItem = inputArea.getByTestId(BUTTON_ADD_ITEM_ID);
+    userEvent.click(buttonAddItem);
+
+    expect(alertSpy).toHaveBeenCalledTimes(1);
+
+    const emptyFields =
+      "Data inválida!\nCategoria inválida!\nTítulo vazio!\nValor inválido!";
+
+    expect(alertSpy).toHaveBeenCalledWith(emptyFields);
+    alertSpy.mockRestore();
+  });
+
+  it("should not be possible to view items registered in other months in the current month.", () => {
+    const inputArea = within(screen.getByTestId(INPUT_AREA_ID));
+    const dataInputArea = inputArea.getByTestId(INPUT_DATA_ITEM_ID);
+    const categInput = inputArea.getByTestId(INPUT_CATEG_ITEM_ID);
+    const titleInput = inputArea.getByTestId(INPUT_TITLE_ITEM_ID);
+    const valueInput = inputArea.getByTestId(INPUT_VALUE_ITEM_ID);
+    const buttonAddItem = inputArea.getByTestId(BUTTON_ADD_ITEM_ID);
+
+    const dateTest = "2023-06-09";
+    const dateTestTable = "09/06/2023";
+    const typeCateg = "Aluguel";
+    const msgTitle = "Apartamento";
+    const valorInput = "800";
+
     userEvent.type(dataInputArea, dateTest);
     userEvent.selectOptions(categInput, typeCateg);
     userEvent.type(titleInput, msgTitle);
     userEvent.type(valueInput, valorInput);
     userEvent.click(buttonAddItem);
-  
-    expect(window.alert).toHaveBeenCalledWith('Data inválida!');
+
+    const tableArea = within(screen.getByTestId(TABLE_AREA_ID));
+
+    expect(tableArea.queryByText(dateTestTable)).not.toBeInTheDocument();
+    expect(tableArea.queryByText(msgTitle)).not.toBeInTheDocument();
+    expect(tableArea.queryByText(`R$ ${valorInput}`)).not.toBeInTheDocument();
+    expect(tableArea.queryByText(typeCateg)).not.toBeInTheDocument();
   });
 
-  it("should not be possible to insert an item without a category.", () => {
+  it("should not be possible to view items registered in the current month in the others months.", () => {
+    const inputArea = within(screen.getByTestId(INPUT_AREA_ID));
+    const dataInputArea = inputArea.getByTestId(INPUT_DATA_ITEM_ID);
+    const categInput = inputArea.getByTestId(INPUT_CATEG_ITEM_ID);
+    const titleInput = inputArea.getByTestId(INPUT_TITLE_ITEM_ID);
+    const valueInput = inputArea.getByTestId(INPUT_VALUE_ITEM_ID);
+    const buttonAddItem = inputArea.getByTestId(BUTTON_ADD_ITEM_ID);
+
+    const infoArea = within(screen.getByTestId(INFO_AREA_ID));
+    const btnextMonth = infoArea.getByTestId(BUTTON_NEXT_MONTH_ID);
+
+    const dateTest = "2023-05-09";
+    const dateTestTable = "09/05/2023";
```

Comment:
Here I think you should use Date.now (if you create the function I suggested on the beginning it would be even better!) ) to get the current date as you did on the first tests, cause the date is hard-coded an on June this test would be invalid